dockerfiles/Dockerfile

```diff
@@ -0,0 +1,83 @@
+    usermod -a -G hadoop $USER
+    
+
+
+# Download HADOOP
+RUN --mount=type=secret,id=wgetrc,dst=/root/.wgetrc \
+    mkdir -p /srv/hops && \
+    wget https://nexus.hops.works/repository/hopshadoop/hops-$HADOOP_VERSION_EE.tgz && \
+    tar -C /srv/hops/ -zxf hops-$HADOOP_VERSION_EE.tgz && \
+    ln -s /srv/hops/hadoop-$HADOOP_VERSION_EE /srv/hops/hadoop && \
+    rm -rf /srv/hops/hadoop/etc && \
+    rm -rf /srv/hops/hadoop/sbin && \
+    rm hops-$HADOOP_VERSION_EE.tgz
+
+# Make sure you build with --platform linux/amd64
+
+# /usr/local/openjdk-8/bin/java
+ENV JAVA_HOME=/usr/local/openjdk-8
+RUN mkdir -p /usr/lib/jvm &&  ln -s $JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+ENV HADOOP_HOME=/srv/hops/hadoop
+ENV HADOOP_CONF_DIR="$HADOOP_HOME/conf"
+
+# Download hive
+# https://archiva.hops.works/repository/Hops/io/hops/hive/hive-packaging/3.0.0.13.3/hive-packaging-3.0.0.13.3-bin.tar.gz
+RUN wget https://archiva.hops.works/repository/Hops/io/hops/hive/hive-packaging/$HIVE_VERSION/hive-packaging-$HIVE_VERSION-bin.tar.gz && \
+    tar -C /srv/hops/ -zxf hive-packaging-$HIVE_VERSION-bin.tar.gz && \
+    ln -s /srv/hops/apache-hive-$HIVE_VERSION-bin /srv/hops/hive && \
+    rm hive-packaging-$HIVE_VERSION-bin.tar.gz
+
+
+ENV HIVE_HOME=/srv/hops/hive
+ENV PATH="/srv/hops/super_crypto/hive:$HIVE_HOME/bin:/opt/venv/bin:$HADOOP_HOME/bin:$PATH"
+
+
+# Download extra jars
+ENV HOPSWORKS_JARS=/srv/hops/hopsworks-jars
+RUN mkdir -p ${HOPSWORKS_JARS}
+# https://repo.hops.works/master/prometheus/jmx_prometheus_javaagent-0.12.0.jar
+RUN wget -O ${HOPSWORKS_JARS}/jmx_prometheus_javaagent-${PROMETHEUS_EXPORTER_VERSION}.jar https://repo.hops.works/master/prometheus/jmx_prometheus_javaagent-${PROMETHEUS_EXPORTER_VERSION}.jar
+# https://repo.hops.works/master/mysql-connector-java-8.0.21-bin.jar
+RUN wget -O ${HOPSWORKS_JARS}/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}-bin.jar https://repo.hops.works/master/mysql-connector-java-${MYSQL_CONNECTOR_VERSION}-bin.jar
+
+RUN wget -O ${HOPSWORKS_JARS}/hudi-hadoop-mr-bundle-$HUDI_BUNDLE_VERSION.jar  http://archiva.hops.works/repository/Hops/io/hops/hudi/hudi-hadoop-mr-bundle/$HUDI_BUNDLE_VERSION/hudi-hadoop-mr-bundle-$HUDI_BUNDLE_VERSION.jar
+
+
+ENV METASTORE_XMX=2g
+ENV METASTORE_XMS=2g
+ENV HOPSWORKS_JARS=/srv/hops/hopsworks-jars
+ENV HIVE_METASTORE_HADOOP_OPTS="-Xmx${METASTORE_XMX} -Xms${METASTORE_XMS} -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=65 -XX:+UseCMSInitiatingOccupancyOnly -javaagent:${HOPSWORKS_JARS}/jmx_prometheus_javaagent-${PROMETHEUS_EXPORTER_VERSION}.jar=18002:${HIVE_HOME}/conf/hivemetastore.yaml -Djdk.tls.ephemeralDHKeySize=2048 -Dlog4j2.formatMsgNoLookups=true"
+ENV HIVE_AUX_JARS_PATH=$HOPSWORKS_JARS
+# ENV HIVESERVER2_HADOOP_OPTS='-Djava.io.tmpdir=/tmp/$HIVE_USER'
+
+
+RUN mkdir -p /home/$USER
+RUN chown $USER:$GROUP /home/$USER
+
+WORKDIR /home/$USER
+COPY dockerfiles/entrypoint.sh entrypoint.sh
+RUN chown $USER:$GROUP  entrypoint.sh 
+RUN chown -R $USER:$GROUP $HIVE_HOME
```

Comment:
I think everything should be user hive, and group hadoop 
```
root@ip-172-16-4-136:/srv/hops/apache-hive# ls -la
total 236
drwxrwx--- 12 hive hadoop   4096 Oct 13 14:45 .
drwxr-xr-x 39 root hadoop   4096 Oct 13 14:54 ..
-rw-r--r--  1 hive hadoop      0 Oct 13 14:02 .hive_extracted_3.0.0.13.2
-rw-r--r--  1 hive hadoop  20798 Sep 10 15:53 LICENSE
-rw-r--r--  1 hive hadoop    230 Sep 10 15:53 NOTICE
-rw-r--r--  1 hive hadoop 143769 Sep 10 15:53 RELEASE_NOTES.txt
drwxr-xr-x  3 hive hadoop   4096 Oct 30 22:44 bin
drwxr-xr-x  2 hive hadoop   4096 Oct 13 14:02 binary-package-licenses
drwxr-xr-x  2 hive hadoop   4096 Feb  1 18:32 conf
drwxr-x---  2 hive hadoop   4096 Oct 13 14:47 consul
drwxr-xr-x  4 hive hadoop   4096 Oct 13 14:02 examples
drwxr-xr-x  7 hive hadoop   4096 Oct 13 14:02 hcatalog
drwxr-xr-x  2 hive hadoop   4096 Oct 13 14:02 hopsworks-jars
drwxr-xr-x  2 hive hadoop   4096 Oct 13 14:02 jdbc
drwxr-xr-x  4 hive hadoop  20480 Oct 13 14:02 lib
lrwxrwxrwx  1 hive hadoop     36 Oct 13 14:45 logs -> /srv/hopsworks-data/apache-hive/logs
drwxr-xr-x  4 hive hadoop   4096 Oct 13 14:02 scripts
root@ip-172-16-4-136:/srv/hops/apache-hive# su hive
hive@ip-172-16-4-136:/srv/hops/apache-hive$ id
uid=1516(hive) gid=1234(hadoop) groups=1234(hadoop),1505(metaserver)
hive@ip-172-16-4-136:/srv/hops/apache-hive$ groups
hadoop metaserver
hive@ip-172-16-4-136:/srv/hops/apache-hive$
```

Reply:
Fixed